payments/schema/types.py

```diff
@@ -225,9 +228,12 @@ class Meta:
         )
 
     @classmethod
-    @view_permission_required(Order)
-    def get_queryset(cls, queryset, info):
-        return super().get_queryset(queryset, info)
+    @login_required
+    def get_node(cls, info, id):
+        node = super().get_node(info, id)
+        return return_node_if_user_has_permissions(
+            node, info.context.user, Order, CustomerProfile
```

Comment:
Do we need to check lease permissions too?

Reply:
Added